src/libs/kata-types/src/config/agent.rs

```diff
@@ -75,6 +75,14 @@ pub struct Agent {
     pub container_pipe_size: u32,
 }
 
+fn default_server_port() -> u32 {
+    1024
+}
+
+fn default_log_port() -> u32 {
+    1025
+}
+
 fn default_dial_timeout() -> u32 {
     // 10ms
```

Comment:
We don't need to comment the value, just the units:

```suggestion
    // ms
```

Reply:
Thanks for your suggestions!